src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,39 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, tabContent, onTabSelected }) => {
+  const handleTabClick = (tabId, content) => {
+    onTabSelected(tabId, content);
+  };
```

Comment:
Duplicate handler. You don't need to create `handleTabClick` function as it doesn't do any additional actions.